src/utils/zcl_abapgit_abap_language_vers.clas.abap

```diff
@@ -5,34 +5,51 @@ CLASS zcl_abapgit_abap_language_vers DEFINITION
 
   PUBLIC SECTION.
 
+    CONSTANTS c_any_abap_language_version TYPE zif_abapgit_aff_types_v1=>ty_abap_language_version VALUE '*'.
     CONSTANTS c_feature_flag TYPE string VALUE 'ALAV'.
 
+    METHODS constructor
+      IMPORTING
+        !io_dot_abapgit TYPE REF TO zcl_abapgit_dot_abapgit.
```

Comment:
can this be changed to `zif_abapgit_dot_abapgit` instead?

Reply:
`zif_abapgit_dot_abapgit` contains only constants. so that would be a bigger change

Reply:
yea, okay